astropy/convolution/tests/test_convolve_models.py

```diff
@@ -8,19 +8,19 @@
 
 from astropy.convolution.convolve import convolve, convolve_fft, convolve_models
 from astropy.modeling import fitting, models
-from astropy.utils.compat.optional_deps import HAS_SCIPY  # noqa
+from astropy.utils.compat.optional_deps import HAS_SCIPY  # noqa: F401
 from astropy.utils.misc import NumpyRNGContext
 
 
 class TestConvolve1DModels:
-    @pytest.mark.parametrize('mode', ['convolve_fft', 'convolve'])
+    @pytest.mark.parametrize('mode', [convolve_fft, convolve])
     @pytest.mark.skipif('not HAS_SCIPY')
     def test_is_consistency_with_astropy_convolution(self, mode):
         kernel = models.Gaussian1D(1, 0, 1)
         model = models.Gaussian1D(1, 0, 1)
-        model_conv = convolve_models(model, kernel, mode=mode)
+        model_conv = convolve_models(model, kernel, mode=mode.__name__)
         x = np.arange(-5, 6)
-        ans = eval(f"{mode}(model(x), kernel(x))")
+        ans = mode(model(x), kernel(x))
```

Comment:
@nstarman is this better? Does everyone else agree?

Reply:
💯